### PR TITLE
boards: Added board file of Ophelia-I

### PR DESCRIPTION
**Pull request by mah-eiSmart**
Add board file of radio module Ophelia-I

Signed-off-by: Matthias Hauser <Matthias.Hauser@we-online.de>

### CODE REVIEW

File: boards/arm/we_ophelia1ev_nrf52805/we_ophelia1ev_nrf52805.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+identifier: we_ophelia1ev_nrf52805
+name: we_ophelia1ev_nrf52805
+type: mcu
+arch: arm
+ram: 24
+flash: 192
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
```
Comment (gmarull) [inline boards/arm/we_ophelia1ev_nrf52805/we_ophelia1ev_nrf52805.yaml:13]: please add list of `supported` tags (for tests)

**Review by gmarull — CHANGES_REQUESTED** [review]
All in-tree boards have now been ported to pinctrl, so please do the same here. You can refer to https://github.com/zephyrproject-rtos/zephyr/pull/43926 for more details.

**Review by gmarull — DISMISSED** [review]

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
Please reduce the board image size to <1MB

**Review by gmarull — DISMISSED** [review]

**Comment by jfischer-no** [comment]
`we_ophelia1ev_nrf52805`, will there be other variants of the board or why is it postfixt with nrf52805?

**Comment by mah-eiSmart** [comment]
> `we_ophelia1ev_nrf52805`, will there be other variants of the board or why is it postfixt with nrf52805?

Hey @jfischer-no, 
there will be only one variant of the Ophelia1 EV.  But, the name of the board was chosen to be consistent to comparable boards developed by Nordic Semiconductor. 
BR Matthias

**Comment by jfischer-no** [comment]
> > `we_ophelia1ev_nrf52805`, will there be other variants of the board or why is it postfixt with nrf52805?
> 
> Hey @jfischer-no, there will be only one variant of the Ophelia1 EV. But, the name of the board was chosen to be consistent to comparable boards developed by Nordic Semiconductor. BR Matthias

But that is for reasons, e.g. nrf52840dk_nrf52840 vs [nrf52840dk_nrf52811](https://www.nordicsemi.com/products/nrf52811), but here it makes no sense.

**Comment by mah-eiSmart** [comment]
> > > `we_ophelia1ev_nrf52805`, will there be other variants of the board or why is it postfixt with nrf52805?
> > 
> > 
> > Hey @jfischer-no, there will be only one variant of the Ophelia1 EV. But, the name of the board was chosen to be consistent to comparable boards developed by Nordic Semiconductor. BR Matthias
> 
> But that is for reasons, e.g. nrf52840dk_nrf52840 vs [nrf52840dk_nrf52811](https://www.nordicsemi.com/products/nrf52811), but here it makes no sense.

Ok, got it. The current proposal is at least consistent to other Würth Elektronik boards, like Proteus-II and Proteus-III, and uses same naming conventions as u-blox, for example.

**Comment by mah-eiSmart** [comment]
> Please reduce the board image size to <1MB

Hey @MaureenHelm,
could you please review the changes and merge the branch?
I updated the product image with a smaller one.

Thx a lot, BR Matthias

File: boards/arm/we_ophelia1ev_nrf52805/board.cmake [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=nrf52" "--speed=4000")
```
Comment (MaureenHelm) [inline boards/arm/we_ophelia1ev_nrf52805/board.cmake:4]: This needs to be updated, see #44181

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

**Review by gmarull — APPROVED** [review]

File: boards/arm/we_ophelia1ev_nrf52805/board.cmake (hunk 2 above)
Comment (mah-eiSmart) [reply boards/arm/we_ophelia1ev_nrf52805/board.cmake]: Done! Thx for feedback!

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**